src/client-apps/nuxeo-drive/nuxeo-drive-release-notes.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Release Notes
+description: .
+tree_item_index: 700
+review:
+  comment: ''
+  date: '2020-03-02'
+  status: ok
+toc: true
+---
+
+Welcome to the Release Notes for **Nuxeo Drive 4.5.0**
+
+## Breaking Changes
+
+### Direct Transfer
+
+The Direct Transfer feature has been disabled for the moment; the implementation is currently being reworked to be more stable.  
```

Comment:
The Direct Transfer feature has been disabled by default for the moment. It can be re-enabled by configuration. It is currently being reworked to be more configurable so as to better match our users' use cases.